build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,37 @@
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int-secnet1.
+OVSIntegrationBridge: {{ (.Values.secondaryNetworkOVS).ovsIntBrName | quote }}
+
+# Name of the OpenVSwitch bridge antrea-agent will create for Secondary network specific tunnel needs.
```

Comment:
"for secondary network transport"? I assume we may support non-tunnel network too.